torch/_inductor/config.py

```diff
@@ -949,10 +949,9 @@ class rocm:
     # If empty, the `native` arch is used
     arch: List[str] = []
 
-    # Enable for CDNA3 only for now
+    # Enable the CK backend for CDNA2 and CDNA3 only (for now)
     # Processor name reference: https://llvm.org/docs/AMDGPUUsage.html#processors
-    # Keep it ordered, unordered set can cause spurious inductor cache misses
-    supported_arch: List[str] = ["gfx940", "gfx941", "gfx942"]
+    ck_supported_arch: List[str] = ["gfx90a"] + ["gfx940", "gfx941", "gfx942"]
```

Comment:
super nit - perhaps just ```["gfx90a", "gfx940", "gfx941", "gfx942"]```?

Reply:
Yep. Here I was trying to make it easier to understand which triplet corresponds to which arch without an extra trip to the triplet name reference. I think I can make it more explicit